### PR TITLE
fix(qa): Multi-arch changes for QA tests

### PR DESCRIPTION
**Pull request by pratham-m**
This PR intends to add codes changes to make QA test multi-arch compliant

### CODE REVIEW

**Comment by pratham-m** [comment]
hi @gavin-stackrox @robbycochran kindly review this PR.

File: qa-tests-backend/src/main/groovy/objects/Deployment.groovy [hunk 1]
```diff
@@ -66,9 +66,9 @@ class Deployment {
     }
 
     static final List<String> TEST_IMAGES_TO_IGNORE_FOR_RATE_LIMIT_CHECK = [
-            "busybox:latest",
```
Comment (robbycochran) [inline qa-tests-backend/src/main/groovy/objects/Deployment.groovy:69]: Why remove`busybox:latest` and `nginx:latest` here ? Do the tests hit rate limit checks on quay.io?

**Review by robbycochran — CHANGES_REQUESTED** [review]

File: qa-tests-backend/src/main/groovy/objects/Deployment.groovy (hunk 1 above)
Comment (pratham-m) [reply qa-tests-backend/src/main/groovy/objects/Deployment.groovy:69]: Specifying just `busybox:latest` and `nginx:latest` would result in pulling the images from `docker.io` by default, which results in pull rate limit error sometimes. Hence explicitly specifying the full image from `quay.io`.

Comment (robbycochran) [reply qa-tests-backend/src/main/groovy/objects/Deployment.groovy:69]: The list of images `TEST_IMAGES_TO_IGNORE_FOR_RATE_LIMIT_CHECK` is not used for pulling, it is used when `setImage` is called. Removing `busybox:latest` and `nginx:latest` here changes behavior of that warning and I don't see the need for adding `quay.io`.

**Review by robbycochran — CHANGES_REQUESTED** [review]

File: qa-tests-backend/src/main/groovy/objects/Deployment.groovy (hunk 1 above)
Comment (jdao-rh) [reply qa-tests-backend/src/main/groovy/objects/Deployment.groovy]: changes reverted

**Pull request merged**